src/applications/mhv-secure-messaging/sass/compose.scss

```diff
@@ -11,13 +9,13 @@
   va-select.composeSelect::part(select) {
     max-width: 100%;
   }
-  @media (min-width: $medium-screen) {
+  @media (min-width: var(--medium-screen)) {
```

Comment:
I am not seeing these apply correctly, while testing in the RI here. For example: this label should get `max-width:100%` on `min-width: var(--medium-screen)`. But it does not and not referencing anywhere in styles, meaning it is not overridden. This is just one example, seeing same thing in other places on `mhv-secure-messaging` styling.  Please advise.

![image](https://github.com/user-attachments/assets/5294b993-308e-49a5-b65e-47e3626c9c5b)